basics/a30.go

```diff
@@ -0,0 +1,17 @@
+// Program exploring signed and unsigned integers
+package main
+
+import "fmt"
+
+func main() {
+	var a uint8
+	a = 255
+	fmt.Printf("var a has value %d and type %T\n", a, a)
```

Comment:
This does not get printed. Is that expected?

Reply:
yes, as further down the code there is an overflow that's occurring.